MAINTAINERS.md

```diff
@@ -0,0 +1,65 @@
+# Maintainers
+
+This document contains list of present maintainers, their role and selection process.
+
+## Current Maintainers
+
+|   Username    |    Focus Area   |
+| ------------- | -------------   |
+|  MarcoFalke   | General, QA     | 
+|  fanquake     | General, Build  |
+|  hebasto      | General, UI/UX  |
+|  achow101     | General, Wallet |
+|  glozow       | General, Mempool|
+
+## Responsibilities and roles
+
+Project maintainers have commit access and are responsible for merging patches 
+from contributors. They perform a janitorial role merging patches that the team 
+agrees should be merged. They also act as a final check to ensure that patches 
+are safe and in line with the project goals. The maintainers’ role is by agreement 
+of project contributors.
+
+## Self Merges
+
+In general, it is considered best practice for maintainers to avoid merging their 
+own changes directly into the codebase. This is because merging your own code can 
+create a potential conflict of interest, where a maintainer's personal interests 
+may override the best interests of the project.
+
+However, it is acceptable for maintainers to merge their own changes if pull request
+has enough reviews and no other maintainer is available to merge.
+
+## Nomination and Selection of Maintainers
+
+The current maintainers or contributors can open a 'call for maintainers' issue. 
```

Comment:
I don't think a single maintainer in the past has been picked based on such a thread, so it seems weird to start prescribing it now and mentioning it seems misleading.

Reply:
Removed 'call for maintainers' section and replaced by nomination in IRC channel. However, it was suggested by @ariard as an improvement that I agree with and could be discussed in next IRC meeting. It can be added later if enough contributors agree with it.